src/fu-progressbar.c

```diff
@@ -103,6 +105,7 @@ fu_progressbar_refresh (FuProgressbar *self, FwupdStatus status, guint percentag
 	guint i;
 	gboolean is_idle_newline = FALSE;
 	g_autoptr(GString) str = g_string_new (NULL);
+	g_autofree gchar *time_remaining = NULL;
```

Comment:
This can move inside the `if` block, no?

Reply:
fixed